src/components/CompareResults/RevisionRow.tsx

```diff
@@ -66,130 +66,135 @@ function RevisionRow(props: RevisionRowProps) {
 
   const styles = {
     revisionRow: style({
+      display: 'grid',
+      gridTemplateColumns: '2fr 1fr 0.2fr 1fr 1fr 1fr 1fr 1fr 2fr 0.2fr',
```

Comment:
can you please add a comment here to mention this should be kept in sync with the other gridTemplateColumns?

Ideally it would be great to share the CSS style, but I'm not sure how to do this with this CSS-in-JS library, so I'm fine with a comment for now.
Please add a comment in the other location too.